boards/arm/efm32gg_slwstk6121a/efm32gg_slwstk6121a.dts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2019 Interay Solutions B.V.
+ * Copyright (c) 2019 Oane Kingma
+ *
```

Comment:
You should also add your copyright line here.

Reply:
Added.